app/components/Home.jsx

```diff
@@ -8,12 +8,13 @@ import ProductCard from './ProductCard'
 const Home = (props) => (
   <div className="container">
     <Row>
-       <Col s={12} m={6} l={4} className='grid-example' key='1'> <ProductCard /> </Col>
-       <Col s={12} m={6} l={4} className='grid-example' key='2'> <ProductCard /> </Col>
-       <Col s={12} m={6} l={4} className='grid-example' key='3'> <ProductCard /> </Col>
-       <Col s={12} m={6} l={4} className='grid-example' key='4'> <ProductCard /> </Col>
-       <Col s={12} m={6} l={4} className='grid-example' key='5'> <ProductCard /> </Col>
-       <Col s={12} m={6} l={4} className='grid-example' key='6'> <ProductCard /> </Col>
+      { 
+        props.products && props.products.map(product => (
+          <Col s={4} className='grid-example' key={product.id}>
```

Comment:
this is great, but could you add the s12 m6 and l4 back in? that helps make it responsive.